buildscripts/build_ffmpeg_windows.sh

```diff
@@ -0,0 +1,99 @@
+#!/bin/bash
+
+#    Copyright © 2021 by The qTox Project Contributors
+#
+#    This program is libre software: you can redistribute it and/or modify
+#    it under the terms of the GNU General Public License as published by
+#    the Free Software Foundation, either version 3 of the License, or
+#    (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU General Public License for more details.
+#
+#    You should have received a copy of the GNU General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
```

Comment:
You said:

>I think most of the code is as it was, just split up.

yet you re-licensed from MIT to GPL and violated MIT by not preserving original license and copyright statements.

The MIT license statement has this part:

># The above copyright notice and this permission notice shall be included in
># all copies or substantial portions of the Software.

Please add the MIT license and the original copyright statements.

Reply:
Good catch. This is a mistake, and probably not even legal. I was under the understanding that qtox was all licensed as GPL and didn't notice the license change. My apologies, fixed.